### ISSUE TITLE
Scutellaria parvula

### ISSUE THREAD

**Issue by MichaelOldham** [Canadensys/vascan-data#2331]
[Originally posted on GoogleCode (id 2429) on 2015-01-23]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>

http://data.canadensys.net/vascan/taxon/6458

<b>What data are incorrect or missing?</b>

Consider recognizing var. missouriensis and var. parvula, as is done in most recent floras (e.g. Voss and Reznicek 2012, Haines 2011).  Both are in Ontario and have different morphology, habitat, and distribution.  Var. missouriensis (or var. leonardii) is recognized as a separate species, S. leonardii, by some authors (e.g. Gleason &amp; Cronquist 1991, Weakley et al. 2012).

MIKE OLDHAM

<b>What data are you expecting instead?</b>

<b>If applicable, please provide an authoritative source.</b>

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2015-02-04 21:50Z]

I have been waiting on the FNA draft for such taxa.

Will do it then.

Luc
